src/utilities/createRouterPush.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { expect, test, vi } from 'vitest'
+import { createRouterPush } from '@/utilities/createRouterPush'
+import { resolveRoutes } from '@/utilities/resolveRoutes'
+import { createRouterNavigation } from '@/utilities/routerNavigation'
+import { routes } from '@/utilities/testHelpers'
+
+test('push calls onLocationUpdated', () => {
```

Comment:
I feel like this aligns with our general strategy so far with unit testing, just wanted to checkin with you to see if we think there are additional good tests that we could do here?

Reply:
I thought about additional tests. Even wrote some, but ended up deleting them as they didn't seem very useful and mostly fell under the "test js does what js is supposed to do". I agree though it feels like there should be more tests around this. Any suggestions?

Reply:
nope, just struck me as odd to have a single test for a function that is not tiny.